c++/src/kj/compat/http-test.c++

```diff
@@ -5000,8 +5000,8 @@ KJ_TEST("adapted client/server propagates request exceptions like non-adapted cl
   KJ_EXPECT_THROW_MESSAGE("request_fail", rawClient->request(HttpMethod::POST, "/"_kj, headers));
   KJ_EXPECT_THROW_MESSAGE("request_fail", adaptedClient->request(HttpMethod::POST, "/"_kj, headers));
 
-  KJ_EXPECT_THROW_MESSAGE("websocket_fail", rawClient->openWebSocket("/"_kj, headers));
-  KJ_EXPECT_THROW_MESSAGE("websocket_fail", adaptedClient->openWebSocket("/"_kj, headers));
+  KJ_EXPECT_THROW_MESSAGE("websocket_fail", rawClient->openWebSocket("/"_kj, headers).ignore());
```

Comment:
I suppose you could argue that KJ_EXPECT_THROW itself should mark the result unused...

Reply:
Done, mostly. I had to add one `(void)` cast to a `KJ_EXPECT_THROW_MESSAGE` in a test for `downcast` (src/kj/common-test.c++ line 576) for reasons I'm not entirely clear on. That could probably use a closer look.